go/vt/binlog/binlogplayer/binlog_player.go

```diff
@@ -80,8 +83,8 @@ func (bps *Stats) SetLastPosition(pos mysql.Position) {
 
 // GetLastPosition gets the last replication position.
 func (bps *Stats) GetLastPosition() mysql.Position {
-	bps.lastPositionMutex.RLock()
-	defer bps.lastPositionMutex.RUnlock()
+	bps.lastPositionMutex.Lock()
+	defer bps.lastPositionMutex.Unlock()
 	return bps.lastPosition
 }
 
```

Comment:
"BinlogPlayer is handling" -> "BinlogPlayer handles".

Reply:
Done.